docs/docs/json.md

```diff
@@ -228,6 +229,7 @@ import org.http4s.circe._
 import org.http4s.dsl.io._
 import org.http4s.ember.server._
 import org.http4s.implicits._
+import org.typelevel.log4cats.slf4j._
```

Comment:
importing these will cause warnings, we should use `Slf4jFactory.create[IO]` in app boot.

Reply:
Good point. I bumped logs4cats to `2.6.0` and replaced `org.typelevel.log4cats.slf4j._` with `Slf4jFactory.create`.